rdb.py

```diff
@@ -0,0 +1,124 @@
+# reddit dataframe class
+# wrapper for PMAW to store reddit data into daily dataframes
+
+import os
+from os import listdir
+from os.path import isfile, join
+
+from pmaw import PushshiftAPI
+import pandas as pd
+import datetime as dt
+from datetime import timedelta 
+
+class rdb:
```

Comment:
Can you change this name to something more descriptive, e.g. `RedditDB` would be better